packages/lib-react-components/src/MetaTools/MetaTools.jsx

```diff
@@ -0,0 +1,76 @@
+import { Box } from 'grommet'
+import { bool, func, objectOf, string } from 'prop-types'
+
+import FavoritesIconButton from '../FavoritesIconButton'
+import CollectIconButton from '../CollectIconButton'
+import ShareIconButton from '../ShareIconButton'
+import InvertIconButton from '../InvertIconButton'
+import ImageIconButton from '../ImageIconButton'
+
+const DEFAULT_LOCATION = {}
+
+function MetaTools({
+  invert = false,
+  location = DEFAULT_LOCATION,
```

Comment:
This expects a raw location object, i.e. `{ 'image/jpeg': 'https://panoptes-uploads.zooniverse.org/123456.jpeg' }` as opposed to a ["processed" location object like what the FlipbookControls](https://github.com/zooniverse/front-end-monorepo/pull/7006/files#diff-3053c4cd1f73bcfde5dfbec9fc2717980ec74b7f1707ab4f8079df7cfe60a52bR78) expects. I think the raw location object is preferable, but open to suggestion otherwise.

Reply:
Good suggestion, I've added some related comments.